SourceCode2Pdf/PythonScriptsExecution/PythonScriptsExecution.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using IronPython.Hosting;
+using Microsoft.Scripting.Hosting;
+
+namespace SourceCode2Pdf.PythonScriptsExecution
+{
+    public class PythonScriptsExecution<T> : PythonScriptExecutor<T>
+    {
+        public T Execute(string script, string resultingVariable)
+        {
+            ScriptEngine pythonScript = Python.CreateEngine();
+            ScriptScope scope = pythonScript.CreateScope();
+            pythonScript.ExecuteFile(script);
```

Comment:
why does this method called ExecuteFile, did you checked that it can exec scripts that stores in script var?

Reply:
yes